artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/impl/TopologyMemberImpl.java

```diff
@@ -154,7 +154,7 @@ public URI toBackupURI() {
 
    @Override
    public String toString() {
-      return "TopologyMember[id = " + nodeId + ", connector=" + connector + ", backupGroupName=" + backupGroupName + ", scaleDownGroupName=" + scaleDownGroupName + "]";
+      return "TopologyMember[id=" + nodeId + ", connector=" + connector + ", backupGroupName=" + backupGroupName + ", scaleDownGroupName=" + scaleDownGroupName + "]";
```

Comment:
Is this change on purpose?

Reply:
yes, though cosmetic. the other items in the same message do not have spacing around the "="